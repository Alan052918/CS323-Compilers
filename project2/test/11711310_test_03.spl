int foo() {
  1 = 12;
  return a;
}

int main() {
  foo();
}
