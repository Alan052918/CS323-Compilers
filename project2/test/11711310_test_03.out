Error type 6 at Line 2: rvalue on the left side of assignment operator
Error type 1 at Line 3: undefined variable: a
